app/config/routing/site.yml

```diff
@@ -10,6 +10,12 @@ company_membership:
   options:
     sitemap: true
 
+member_membership:
+  path: /adherer/member
```

Comment:
C'est du joli Franglais. Il faut choisir et ce n'est pas facile ;-)

Reply:
J'ai suivi le mouvement :p

Je vois de tout dans le code du coups je me demande, actuellement, on vise plus le full anglais ou le full français en terme de : 
* URL
* Code

Reply:
Dans #1139 il y avait la proposition d'utiliser `/adherer/particulier` : https://github.com/afup/web/issues/1139#issuecomment-2156805849

Reply:
Oups ! Pardon c'était passé à la trappe ! Donc url en fr, c'est noté ! Je fais le changement dans la journée :)

Reply:
Pas de soucis ça sert à ça la review :)

Reply:
La journée a été longue mais c'est fait :')